### PR TITLE
[zh-cn]sync pods/pod-lifecycle.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md

### CODE REVIEW

**Comment by tengqm** [comment]
@my-git9 Any thoughts on moving this forward?

**Comment by my-git9** [comment]
> @my-git9 Any thoughts on moving this forward?

Sorry for the delay

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 1]
```diff
@@ -977,27 +977,51 @@ documentation for
...
+3. are terminating pods, bound to a non-ready node tainted with [`node.kubernetes.io/out-of-service`](/docs/reference/labels-annotations-taints/#node-kubernetes-io-out-of-service), when the `NodeOutOfServiceVolumeDetach` feature gate is enabled.
+
+When the `PodDisruptionConditions` feature gate is enabled, along with
+cleaning up the pods, PodGC will also mark them as failed if they are in a non-terminal
+phase. Also, PodGC adds a pod disruption condition when cleaning up an orphan
+pod (see also:
+[Pod disruption conditions](/docs/concepts/workloads/pods/disruptions#pod-disruption-conditions)).
+-->
+此外，PodGC 会清理满足以下任一条件的所有 Pod：
+1. 是孤儿 Pod - 绑定到不再存在的节点，
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:1014]: I think "which satisfy any of ...  1. are ..." done need to keep "是" in Mandarin context.

```suggestion
1. 孤儿 Pod - 绑定到不再存在的节点
```

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 2]
```diff
@@ -977,27 +977,51 @@ documentation for
...
+2. 是计划外的终止的 Pod
+3. 是终止过程中的 Pod，当启用 `NodeOutOfServiceVolumeDetach` 功能门控时，
```
Comment (Sea-n) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:1016]: @tengqm already spotted that feature gate is "特性門控" instead of "功能門控".

**Pull request merged**